src/main/java/Babe.java

```diff
@@ -0,0 +1,448 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import exception.NoDescriptionException;
+import exception.NonsenseInputException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+/**
+ * <h1> Hi Babe! </h1>
+ * Babe is a chat bot created as part of the individual project (iP) under the course CS2109T.
+ * It is a duplicate of Duke with some personal flair.
+ *
+ * @author Shan Hern Hng
+ * @version 1.0
+ * @since 17 January 2023
+ */
+public class Babe {
+
+
+    /**
+     * A string input from user.
+     */
+    private ArrayList<String> userInput = new ArrayList<>();
+
+    /**
+     * Length of user input.
+     */
+    private int userInputLen = 0;
+
+    /**
+     * List of Tasks Babe received from the user.
+     */
+    private ArrayList<Task> memory = new ArrayList<>();
```

Comment:
Here more appropriate to name it as "memories" to show it a collection of items?